mathics/builtin/numbers/algebra.py

```diff
@@ -1106,7 +1120,16 @@ def apply(self, expr, evaluation):
         return variables
 
 
+# This Builtin belongs to mathics.builtin.list.eol
```

Comment:
Do you mind if we just move it there?

Reply:
Go ahead. I did not do here to make this PR shorter and clearer